### ISSUE TITLE
How to disable zooming on click 

### ISSUE THREAD

**Issue by mimre73** [thuanmb/react-mapbox-gl-cluster#16]
i want to **disable zooming on click** and only get properties of cluster when user click on cluster how can do that?

**Comment by thuanmb** [comment]
I haven't supported this property. Let me support it and update the status with you ASAP.

**Comment by thuanmb** [comment]
The `clusterClickEnabled` flag is available at `react-mapbox-gl-cluster@1.11.0`, the usage is:

```
<ReactMapboxGlCluster
    data={data}
    clusterClickEnabled={false}
/>

```
Regards!
